packages/stylelint-copyright/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "stylelint-copyright",
```

Comment:
Indentation is off here. I recommend you use VS Code and install the Prettier plugin to autoformat your JS files upon save.